.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = E117,E501,E722,W191
```

Comment:
would really like to see E117/E501/W191 out of this list since it can be cleaned up in one go and configured in text/code editors and IDEs easily.. :)